manifests/plugin/python/module.pp

```diff
@@ -0,0 +1,34 @@
+# Single module definition
+define collectd::plugin::python::module (
+  $config,
+  $script_source = undef,
+  $modulepath    = $collectd::plugin::python::modulepath,
+  $module        = $title,
+  $ensure        = 'present',
+  $order         = '50',
```

Comment:
Nope, not really. AFAIK all the Python plugins are loaded together and the order is not important.